pkg/plugins/policies/core/jsonpatch/merge_test.go

```diff
@@ -67,9 +67,17 @@ var _ = Describe("Json Patch merge", func() {
 		}
 
 		// when
-		_, err := jsonpatch.MergeJsonPatchAny(util_proto.MustMarshalAny(hcm), patches)
+		hcmAny := util_proto.MustMarshalAny(hcm)
+		mergedHcmAny, err := jsonpatch.MergeJsonPatchAny(hcmAny, patches)
```

Comment:
how does it create camelCase?

Reply:
I've used a library but if we think it is error prone we could try do regex and match->replace https://github.com/gobeam/stringy/blob/master/stringy.go#L112